Cargo.toml

```diff
@@ -126,7 +126,7 @@ regex = "1.11.1"
 reqwest = { version = "0.12.15", default-features = false }
 reqwest-middleware = "0.4.2"
 reqwest-retry = "0.7.0"
-resolvo = { version = "0.10.0" }
+resolvo = "0.10.0"
```

Comment:
`0.10.1` also seems to be available. Maybe we can bump to that in the lock-file.